src/main/java/bot/commands/utils/OptionsCommand.java

```diff
@@ -43,5 +43,8 @@ protected void execute(CommandEvent event) {
         String guildId = event.getGuild().getId();
         String optionName = arguments[0].toLowerCase(Locale.ROOT);
         OptionEntity optionEntity = optionEntityRepository.findByServerIdAndName(guildId, optionName);
+        if (optionEntity != null ){
```

Comment:
?? not sure but it works

Reply:
If we supplement the conditions by passing the argument, it may not work, I think it's some issue for investigation